common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -795,6 +797,61 @@ def test_courseware_nav(self):
         self.course_nav.go_to_section('Test Section 2', 'Test Subsection 3')
         self.assertTrue(self.course_nav.is_on_section('Test Section 2', 'Test Subsection 3'))
 
+    # def test_course_home(self):
```

Comment:
Is this necessary anymore? It looks like this should be covered by the test below this (and they have the same name anyway).